SS_param.tpl

```diff
@@ -17,7 +17,8 @@ PARAMETER_SECTION
   // clang-format on
   // set the filename to all ADMB output files to "ss.[ext]"
   ad_comm::adprogram_name = "ss";
-  echoinput << "now in PARAMETER_SECTION " << endl;
+  echoinput << "Begin setting up parameters" << endl;
+  cout << "Setting up parameters... ";
```

Comment:
Why are these statements on line 20 and 21 different? If you keep line 21 as is, I think there needs to be a space after "parameters" before the l-dots.